core/domain/user_jobs_one_off.py

```diff
@@ -335,48 +335,48 @@ def reduce(key, stringified_values):
         pass
 
 
-class UserRolesMigrationOneOffJob(jobs.BaseMapReduceJobManager):
-    """A one time job to attach a new field (representing role of user) to the
-    UserSettingsModel. The job will load all existing users from the datastore,
-    look for the role to assign them and then store them back thus updating the
-    table schema and adding appropiate roles to users.
-    """
-    @classmethod
-    def entity_classes_to_map_over(cls):
-        return [user_models.UserSettingsModel]
-
-    @staticmethod
-    def map(user_model):
-        admin_usernames = config_domain.Registry.get_config_property(
-            'admin_usernames')
-        moderator_usernames = config_domain.Registry.get_config_property(
-            'moderator_usernames')
-        banned_usernames = config_domain.Registry.get_config_property(
-            'banned_usernames')
-        collection_editors = config_domain.Registry.get_config_property(
-            'collection_editor_whitelist')
-
-        user_roles = [feconf.ROLE_ID_EXPLORATION_EDITOR]
-        if user_model.username in admin_usernames.value:
-            user_roles.append(feconf.ROLE_ID_ADMIN)
-        if user_model.username in moderator_usernames.value:
-            user_roles.append(feconf.ROLE_ID_MODERATOR)
-        if user_model.username in banned_usernames.value:
-            user_roles.append(feconf.ROLE_ID_BANNED_USER)
-        if user_model.username in collection_editors.value:
-            user_roles.append(feconf.ROLE_ID_COLLECTION_EDITOR)
-        user_role = role_services.get_max_priority_role(user_roles)
-
-        try:
-            user_services.update_user_role(user_model.id, user_role)
-            yield ('success', 1)
-        except Exception as e:
-            logging.error('Exception raised: %s' % e)
-            yield (user_model.username, unicode(e))
-
-    @staticmethod
-    def reduce(key, value):
-        if key == 'success':
-            yield(key, len(value))
-        else:
-            yield(key, value)
+# class UserRolesMigrationOneOffJob(jobs.BaseMapReduceJobManager):
```

Comment:
can I remove both the job and the test? Since the job has run and it includes code that was there for keeping sync of two systems, like ROLE_SYNC_DICT, get_max_priority_role in role_services that has been removed.